ex1/dates_times.py

```diff
@@ -30,15 +30,47 @@
 Exercises for using the datetime and the calendar module
 '''
 
+from calendar import isleap
+from datetime import date,time,datetime,timedelta
+
 # Define a function named last_of_month that takes an argument dt of type date
 # and returns a date object representing the last day of the month dt was in.
 
+def last_of_month(dt):
```

Comment:
The [calendar.monthrange](https://docs.python.org/3.4/library/calendar.html#calendar.monthrange) function would have made your live easier.